data/viveksinra.json

```diff
@@ -0,0 +1,116 @@
+{
+	"name": "Vivek Singh",
+	"type": "personal",
+	"displayStatsPublic": true,
+	"bio": "Full Stack Developer | Founder of **Softechinfra** | **Qualifier.co.in** CEO",
+	"avatar": "https://avatars.githubusercontent.com/u/37905082?v=4",
```

Comment:
In the latest version you do not need to include avatar: as it will be automatically fetched from your profile. Please do remove it and let me know if you face any issue. 
the updated docs are available at https://linkfree.eddiehub.io/docs/quickstart